<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LockOperateDialog</class>
 <widget class="QDialog" name="LockOperateDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>95</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>锁车控制 - 通道 0</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="bindButton">
       <property name="text">
        <string>绑定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="unbindButton">
       <property name="text">
        <string>解绑</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="unlockButton">
       <property name="text">
        <string>锁车</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="lockButton">
       <property name="text">
        <string>解锁</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>触发CANID：0x</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lockTriggerCanidLineEdit">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>10123456</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>数据：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lockTriggerDataLineEdit">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>00 00 00 00 00 00 00 00</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="assignGPSIDCheckBox">
       <property name="text">
        <string>指定GPSID：0x</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="GPSIDLineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="solidkeyCheckBox">
       <property name="text">
        <string>指定固定密钥：0x</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="solidkeyLineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
